crates/core/c8y_smartrest/src/error.rs

```diff
@@ -1,5 +1,5 @@
 use agent_interface::SoftwareUpdateResponse;
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
```

Comment:
Unused import marked by clippy.

```suggestion
use std::path::PathBuf;
```